benchmarks/search/radius_search.cpp

```diff
@@ -0,0 +1,53 @@
+#include <pcl/io/pcd_io.h>
+#include <pcl/search/organized.h>
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+
+#include <benchmark/benchmark.h>
+
+static void
+BM_OrganizedNeighborSearch(benchmark::State& state, const std::string& file)
+{
+  pcl::PointCloud<pcl::PointXYZ>::Ptr cloudIn(new pcl::PointCloud<pcl::PointXYZ>);
+  pcl::PCDReader reader;
+  reader.read(file, *cloudIn);
+
+  pcl::search::OrganizedNeighbor<pcl::PointXYZ> organizedNeighborSearch;
+
+  double radiusSearchTime = 0;
+
+  for (auto _ : state) {
+    organizedNeighborSearch.setInputCloud(cloudIn);
+
+    int randomIdx = rand() % cloudIn->size();
+    double searchRadius = 1.0 * ((double)rand() / (double)RAND_MAX);
+
+    std::vector<int> k_indices;
+    std::vector<float> k_sqr_distances;
+
+    auto check_time = benchmark::Now();
```

Comment:
The compiler doesn't know `benchmark::Now()`. Unless you have a better idea, I would suggest to simply use the default/automatic way and measure the duration of all functions in the `for` loop.